packages/nitro-protocol/package.json

```diff
@@ -33,15 +33,18 @@
   "scripts": {
     "prepare": "yarn build",
     "build:ci": "yarn build",
-    "test:ci": "yarn test --runInBand",
     "test:typescript": "npx jest --testPathIgnorePatterns='test/contracts' --reporters='default'",
-    "test": "yarn lint:check && yarn clearContracts && npx ganache-then-jest",
-    "deploy": "yarn clearContracts && npx truffle migrate",
+    "test:prepare": "yarn lint:check && yarn deploy:test",
+    "test": "yarn test:prepare && npx jest",
+    "test:ci": "yarn contract:compile && yarn test:prepare && npx jest --ci --bail",
+    "deploy": "npx etherlime deploy --compile=false",
+    "deploy:test": "yarn deploy && npx etherlime deploy --compile=false --file=./deployment/deploy-test.js",
+    "startGanache": "npx etherlime ganache --verbose > ganache.log",
     "build:typescript": "tslint --project . && tsc -b",
-    "build:truffle": "npx truffle compile",
+    "contract:compile": "npx etherlime compile --runs=1 --buildDirectory=./build/contracts",
```

Comment:
The value of 1 for `--runs` is just something I tried at some point to turn on the optimizer. I am not actually sure if that is the correct value to use.